addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -987,18 +987,25 @@ export class Wysiwyg extends Component {
         this.odooEditor.historyReset();
     }
     /**
-     * Saves the content.
+     * Saves the content or the given editable.
      *
      * @param {boolean} [reload=true]
+     * @param {Object} [editable=false] Specific editable to save
      * @returns {Promise}
      */
-    async saveContent(reload = true) {
+    async saveContent(reload = true, editable = false) {
```

Comment:
The commit message explains what it does, but not why it is needed. Therefore, it looks fishy to me. We have had the same kind of issues with the `cleanForSave` of snippet options before and we worked around the issue rather than changing the api of snippet options. If we change this one here then we might want to change the snippet options api as well, or even other parts of the code that are locked to a predetermined node so that they can work on any arbitrary node.

Reply:
This changes have been made for this commit: https://github.com/odoo/enterprise/pull/47089/commits/c9940160ac363a30ada0d7f68e38e9000f98d8b5

We need to save a clone of the report editor, like that the current report editor is not impacted by the cleanForSave if the save operation failed further in the process.